.devcontainer/docker-compose.yml

```diff
@@ -2,11 +2,15 @@ version: "3.8"
 
 services:
   devcontainer:
+    env_file: ./.env
```

Comment:
This works for me using the `environment` block without a default value, even though I haven't set the `INTEGREAT_CMS_CONFIG` variable in my shell. If we need a default value, we can also set it directly in the Compose file using the `:-` notation.

For example:
```yaml
environment:
  - INTEGREAT_CMS_CONFIG=${INTEGREAT_CMS_CONFIG}
```

or with default value

```yaml
environment:
  - INTEGREAT_CMS_CONFIG=${INTEGREAT_CMS_CONFIG:-default}
```

Reply:
Thanks for the hint, A thought I have is if it would make sense to have the `.env` file to have one place to pas environment variables in general in case someone would use it that way. But I don't really know how far a devcontainer should be individualized that way.
It should be fine without a default value. If the variable isn't set, it will automatically be set to blank inside the container.